server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@neohere97 you shouldn't need to duplicate this file, you should be able to share with your char driver.